pulsar-io/kafka-connect-adaptor/pom.xml

```diff
@@ -166,6 +166,19 @@
       </exclusions>
     </dependency>
 
+    <dependency>
+      <groupId>org.apache.kafka</groupId>
+      <artifactId>connect-transforms</artifactId>
+      <version>${kafka-client.version}</version>
+      <scope>test</scope>
```

Comment:
It seems that we should include this in default scope so that default transformations (and later predicates) would be available.

Reply:
The jar file is just over 100kB, so it won't add much overhead to the current connectors.

Reply:
Makes sense. I was overly defensive, it seems 😄

> The connect-transforms package has only been added as a test dependency. This is to keep the Kafka Connect adaptor bundle as lightweight as possible. Connectors that rely on transforms, such as Debezium, typically package their own set of Kafka Connect transforms, making it unnecessary to include them in the adaptor itself.

I will take care of this shortly. Thanks for taking the time to review this change! 🙏